class-00/lab-b/starter-code/app.js

```diff
@@ -108,42 +108,36 @@ const newObject = array => ({
 
 
 
-let sum = function(a, b, c, d) {
-  return a + b + c + d;
-};
+let sum = (a, b, c, d) => ` ${a} + ${b} + ${c} + ${d} ` ;

```

Comment:
You don't need to use `${}`
This will return the string 'a + b + c + d' instead of adding the numbers and returning the sum